packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.controller.js

```diff
@@ -0,0 +1,73 @@
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $anchorScroll,
+    $location,
+    $translate,
+    Alerter,
+    datacenterBackupService,
+  ) {
+    this.$anchorScroll = $anchorScroll;
+    this.$location = $location;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.backup = {};
+    this.alerter = Alerter;
+    this.datacenterBackupService = datacenterBackupService;
+  }
+
+  $onInit() {
+    this.alertMain = 'app.dedicatedClouds.datacenter.backup';
+    this.loader = {
+      updatingCapabilities: false,
+    };
+  }
+
+  scrollToTop() {
+    this.$location.hash('backupMainAlert');
```

Comment:
I'm not sure the component should be aware of the `$location`

Reply:
> This is due to use of `$anchorScroll` I guess. @radireddy I know I did recommend to use `$anchorScroll` in another PR.
> Can you try to do please:

This works and is incorporated in the code @JeremyDec . Thanks a lot!